Resources/Computing/BatchSystems/TimeLeft/ResourceUsage.py

```diff
@@ -19,9 +19,14 @@ def __init__(self, batchSystemName, jobIdEnvVar):
     self.jobID = os.environ.get(jobIdEnvVar)
 
   def getResourceUsage(self):
-    """ Returns a dictionary containing CPUConsumed, CPULimit, WallClockConsumed
-        and WallClockLimit for current slot.  All values returned in seconds.
+    """ Returns S_OK with a dictionary that can contain entries:
+        - CPU: the CPU time consumed since the beginning of the execution for current slot (seconds)
```

Comment:
```suggestion

        - CPU: the CPU time consumed since the beginning of the execution for current slot (seconds)
```
Empty line before bullet list